cs/src/Connections/TunnelRelayTunnelHost.cs

```diff
@@ -558,26 +558,21 @@ private void OnSshClientAuthenticating(object? sender, SshAuthenticatingEventArg
     private async Task StartForwardingExistingPortsAsync(
         SshSession session, bool removeUnusedPorts = false)
     {
+        // Send port-forward request messages concurrently. The client may still handle the
+        // requests sequentially but at least there is no network round-trip between them.
+        var forwardTasks = new List<Task>();
+
         var tunnelPorts = Tunnel!.Ports ?? Enumerable.Empty<TunnelPort>();
         var pfs = session.ActivateService<PortForwardingService>();
         pfs.ForwardConnectionsToLocalPorts = this.ForwardConnectionsToLocalPorts;
         foreach (TunnelPort port in tunnelPorts)
         {
-            try
-            {
-                await ForwardPortAsync(pfs, port, CancellationToken.None);
-            }
-            catch (Exception exception)
-            {
-                Trace.TraceEvent(
-                    TraceEventType.Error,
-                    0,
-                    "Error forwarding port {0} to client: {1}",
-                    port.PortNumber,
-                    exception.Message);
-            }
+            // ForwardPortAsync() catches and logs most exceptions that might normally occur.
+            forwardTasks.Add(ForwardPortAsync(pfs, port, CancellationToken.None));
```

Comment:
Before the change the exceptions from `ForwardPortAsync` were traced and ignored. Now they'd pop up to the caller. Is this intended?

Reply:
Yes - see the comment I added on line 570. The catch here was redundant, unless some unexpected kind of exception occurred, and in that case a crash is probably preferred over ignoring it.